index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Richard Mbuti</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <style></style>
+  </head>
+  <body>
+    <header class="topHeader">
+      <a href="#" class="logo">RK</a>
+      <nav class="menu">
+        <a href="#" class="toggleNav">
+          <img src="img/Union.png" alt="menu_button" />
+        </a>
+      </nav>
+    </header>
+    <br />
+    <br />
+    <section class="backgroundImg">
+      <div class="Personal_info">
+        <h1 class="introName">I'm Richard Mbuti Glad to see you!</h1>
+        <p class="intro">
+          I'm a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don't hestiate to contact
+          me.
+        </p>
+        <h6 class="connect">LET'S CONNECT</h6>
+      </div>
+      <div class="my_links">
+        <ul class="media_links">
+          <li>
+            <a href="#" target="_blank">
+              <i class="fa fa-linkedin"></i>
+            </a>
+          </li>
+          <li>
+            <a href="#" target="_blank">
+              <i class="fa fa-github"></i>
+            </a>
+          </li>
+          <li>
+            <a href="#" target="_blank">
+              <i class="fa fa-twitter"></i>
+            </a>
+          </li>
+          <li>
+            <a href="#" target="_blank">
+              <i class="fa fa-facebook"></i>
+            </a>
```

Comment:
:red_circle: Kindly make sure your icons color match the  Figma Design

Your Design
![isons](https://user-images.githubusercontent.com/26736582/205442759-1cc909b7-ba57-4d28-8225-8ce57a951283.png)

Figma Design
![icons](https://user-images.githubusercontent.com/26736582/205442769-c1867df1-cd47-47b9-97f0-10ad7c2c0a3c.png)